src/types/logging/index.ts

```diff
@@ -0,0 +1 @@
+export * from './Transformers';
```

Comment:
I checked the other `index.ts` files in the rest of sub folders (under `types` folder) and they have a boilerplate commented message in the beginning. So I was wondering if it is a good idea to add it to this file too as shown below : 
```suggestion
// Copyright 2017-2022 Parity Technologies (UK) Ltd.
// This file is part of Substrate API Sidecar.
//
// Substrate API Sidecar is free software: you can redistribute it and/or modify
// it under the terms of the GNU General Public License as published by
// the Free Software Foundation, either version 3 of the License, or
// (at your option) any later version.
//
// This program is distributed in the hope that it will be useful,
// but WITHOUT ANY WARRANTY; without even the implied warranty of
// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
// GNU General Public License for more details.
//
// You should have received a copy of the GNU General Public License
// along with this program.  If not, see <http://www.gnu.org/licenses/>.

export * from './Transformers';
```